geoportailv3/static/js/measure/measuredirective.js

```diff
@@ -134,4 +171,21 @@ app.MeasureController = function($scope, ngeoDecorateInteraction) {
 };
 
 
+/**
+ * @param {ol.Coordinate} coordinates
+ * @return {angular.$q.Promise}
+ * @private
+ */
+app.MeasureController.prototype.getElevation_ = function(coordinates) {
+  var lonlat =
```

Comment:
@elemoine Yes indeed this is not a longitude and latitude. But I chose the name lonlat, because the elevation webservice is using lon and lat as parameter name.
Thus I've used the same convention, in order to show that the coordinates are in the same reference system than the ones used by the webservice.
In your opinion, what would be a better name ? eastnorth ?